docs/streamlit_components.md

```diff
@@ -11,7 +11,9 @@ Streamlit Components let you expand the functionality provided in the base Strea
 - Rendering Python objects having methods that output HTML, such as IPython [`__repr_html__`](https://ipython.readthedocs.io/en/stable/config/integrating.html#rich-display)
 - Convenience functions for commonly-used web features like [GitHub gists and Pastebin](https://github.com/randyzwitch/streamlit-embedcode)
 
-Check out this [Streamlit Components Tutorial video](https://www.youtube.com/watch?v=BuD3gILJW-Q&feature=youtu.be) by Streamlit engineer Tim Conkling to get started:
+Check out these Streamlit Components Tutorial videos by Streamlit engineer Tim Conkling to get started:
+
+[Part 1: Setup and Architecture](https://www.youtube.com/watch?v=BuD3gILJW-Q&feature=youtu.be)
```

Comment:
Nit: These could be headers rather than links, especially since the youtube embed already exists, and has a link back to the Youtube page if they want it